app/services/katello/event_daemon/runner.rb

```diff
@@ -0,0 +1,94 @@
+module Katello
```

Comment:
This is the file formerly known as `event_daemon.rb`

Reply:
![image](https://user-images.githubusercontent.com/1979598/106318395-e46fee80-623d-11eb-99e5-b674a18377f2.png)